wpimath/src/main/java/edu/wpi/first/math/kinematics/ChassisSpeeds.java

```diff
@@ -143,6 +143,51 @@ public static ChassisSpeeds fromFieldRelativeSpeeds(
         robotAngle);
   }
 
+  /**
+   * Converts a user provided robot-relative set of speeds into a field-relative ChassisSpeeds
+   * object.
+   *
+   * @param vxMetersPerSecond The component of speed in the x direction relative to the robot.
+   *     Positive x is the robot's forward.
```

Comment:
This reads weirdly, maybe "towards the robot's front" or "forward relative to the robot"?